doc/release-notes-16884.md

```diff
@@ -0,0 +1,4 @@
+The wallet now by default uses bech32 addresses when using RPC, and creates native segwit
```

Comment:
Drop "when using RPC"?

Reply:
QT already by default does bech32, so I believe this is more accurate?